gax/src/main/java/com/google/api/gax/core/GoogleCredentialsProvider.java

```diff
@@ -62,6 +67,16 @@
 
   @Override
   public Credentials getCredentials() throws IOException {
+    return getCredentials(false, null);
+  }
+
+  /**
+   * If user doesn't provide custom endpoint and scopes, and the credentials is service account
+   * credentials, then we need to create a new ServiceAccountJwtAccessCredentials to use self signed
```

Comment:
rewrite in third person
This doc comment only seems to describe one detail of the method's implementation, not the entire method.